.github/workflows/build-docker-image.yml

```diff
@@ -1,6 +1,9 @@
 name: Build Docker Image
 on:
   pull_request:
+  push:
+    branches:
+      - 'gh-readonly-queue/**'
```

Comment:
Is the double * necessary/significantly different than a single *?

Reply:
Its a glob pattern, so * matches any character other than a slash. ** matches slashes too,